client/landing/stepper/declarative-flow/flows/build/build.ts

```diff
@@ -51,16 +44,13 @@ const build: Flow = {
 			() => ( {
 				eventsProperties: {
 					[ STEPPER_TRACKS_EVENT_SIGNUP_STEP_START ]: {
-						...( isGoalsAtFrontExperiment && {
-							is_goals_first: 'true',
-						} ),
 						...( initialGoals.current.length && {
 							goals: initialGoals.current.join( ',' ),
```

Comment:
This was added here, can probably be removed as well https://github.com/Automattic/wp-calypso/pull/98448